lib/prometheus/business_collector.rb

```diff
@@ -122,6 +128,26 @@ def collect
     @client_50plus_policies.observe(
       policies_50plus_hosts.where('policies.account_id' => client_accounts.select(:id)).count.size
     )
+
+    @total_reports_per_month.observe(
+      TestResult.where(
+        'updated_at >= ? AND updated_at <= ?',
+        DateTime.now.beginning_of_month,
+        DateTime.now.end_of_month
```

Comment:
Nit: I'd rather use an open interval from right and use `DateTime.now.next_month.beginning_of_month`.